tools/benchmark/cmd/encoder.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2015 The etcd Authors
```

Comment:
nit, I would suggest to use 2020 here instead of 2015.

Reply:
done